x/wasm/keeper/handler_plugin_test.go

```diff
@@ -323,16 +384,16 @@ func TestIBCRawPacketHandler(t *testing.T) {
 			assert.Nil(t, evts)
 			require.NotNil(t, data)
 			assert.Len(t, msgResponses, 1)
-			assert.Equal(t, "/cosmwasm.wasm.v1.MsgIBCSendResponse", msgResponses[0][0].TypeUrl)
-
-			expMsg := types.MsgIBCSendResponse{Sequence: 1}
+			// assert.Equal(t, "/cosmwasm.wasm.v1.MsgIBCSendResponse", msgResponses[0][0].TypeUrl)
```

Comment:
There should be a way to get the typeURL of `spec.expResp` via



```
// MessageName returns the fully-qualified proto name for the given message type.
func MessageName(x Message) string {
```

which is in `/Users/me/go/pkg/mod/github.com/cosmos/gogoproto@v1.5.0/proto/properties.go`